### PR TITLE
Plot Peddy het check

### PR DESCRIPTION
**Pull request by aledj2**
New peddy plot for that shows `x=median_depth`, `y=het_ratio`. See #650

### CODE REVIEW

File: multiqc/modules/peddy/peddy.py [hunk 1]
```diff
@@ -205,3 +212,31 @@ def peddy_relatedness_plot(self):
...
+        
+        pconfig = {
+            'id': 'peddy_het_check_plot',
+            'title': 'Peddy Het Check Plot',
+            'xlab': 'median depth',
+            'ylab': 'proportion het calls',
+            }
+    
+        self.add_section (
+            name = 'Peddy Het Check Plot',
```
Comment (ewels) [inline multiqc/modules/peddy/peddy.py]: This section is already under Peddy in the nav, so no need to repeat this again - `Het Check` should be fine.

File: multiqc/modules/peddy/peddy.py [hunk 2]
```diff
@@ -205,3 +212,31 @@ def peddy_relatedness_plot(self):
...
+                data[s_name] = {
+                    'x': d['median_depth_het_check'],
+                    'y': d['het_ratio_het_check']
+                }
+        
+        pconfig = {
+            'id': 'peddy_het_check_plot',
+            'title': 'Peddy Het Check Plot',
+            'xlab': 'median depth',
+            'ylab': 'proportion het calls',
```
Comment (ewels) [inline multiqc/modules/peddy/peddy.py]: Is it worth forcing a y axis scale from 0-1 here? Or would that mask differences that we're looking for?

File: multiqc/modules/peddy/peddy.py [hunk 3]
```diff
@@ -205,3 +212,31 @@ def peddy_relatedness_plot(self):
...
+            if 'median_depth_het_check' in d and 'het_ratio_het_check' in d:
+                # add sample to dictionary with value as a dictionary of points to plot
+                data[s_name] = {
+                    'x': d['median_depth_het_check'],
+                    'y': d['het_ratio_het_check']
+                }
+        
+        pconfig = {
+            'id': 'peddy_het_check_plot',
+            'title': 'Peddy Het Check Plot',
```
Comment (ewels) [inline multiqc/modules/peddy/peddy.py]: Preferred syntax for plot titles is now to have a colon: `Peddy: Het Check`

File: multiqc/modules/peddy/peddy.py (hunk 1 above)
Comment (ewels) [reply multiqc/modules/peddy/peddy.py]: Also, please add at least a `description` and preferably also a longer `helptext` for this section.

File: multiqc/modules/peddy/peddy.py (hunk 2 above)
Comment (aledj2) [reply multiqc/modules/peddy/peddy.py]: Yea, I think 0-1 Y axis would make the differences harder to spot. I have set the y limits to +/- 0.1 from min/max values, but now I think about it I prefer it with the original padding! Will undo this change

**Comment by ewels** [comment]
Looks great, thanks!

**Comment by ewels** [comment]
Hi @aledj2,

A bit late, now that I've merged this, but I'm not completely clear of the use of plotting this against `median_depth`? Would this not be clearer as a simple bar plot of `proportion_het_calls`? I feel like that would be quite a bit easier to read..

Phil

**Pull request merged**